sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -17,13 +17,9 @@ The client requires an instance of [TokenCredential](https://docs.microsoft.com/
 In this samples, we illustrate how to use one derived class: ClientSecretCredential.
 
 ```C# Snippet:DigitalTwinsSampleCreateServiceClientWithClientSecret
-// By using the ClientSecretCredential, a specified application Id can login using a
-// client secret.
-var tokenCredential = new ClientSecretCredential(
-    tenantId,
-    clientId,
-    clientSecret,
-    new TokenCredentialOptions { AuthorityHost = KnownAuthorityHosts.AzureCloud });
+// DefaultAzureCredential supports different authentication mechanisms and determines the appropriate credential type based of the environment it is executing in.
+// It attempts to use multiple credential types in an order till it finds a working credential.
```

Comment:
till = until

Reply:
I know "till" is legit, but we don't use it as often in this context. Just sounds weird.

Reply:
ok :)